lib/apus/adapters/twilio_adapter.ex

```diff
@@ -4,17 +4,58 @@ defmodule Apus.TwilioAdapter do
 
   @behaviour Apus.Adapter
 
-  alias ExTwilio.Message
-
   def deliver(message, config) do
+    params = message |> convert_to_twilio_params(config) |> to_query_string()
+
+    case :hackney.post(uri(config), headers(config), params, options(config)) do
+      {:ok, status, _headers, response} when status > 299 ->
+        {:ok, body} = :hackney.body(response)
+        body = Poison.decode!(body)
+        {:error, body["message"]}
+
+      {:ok, _status, _headers, response} ->
+        {:ok, body} = :hackney.body(response)
+        Poison.decode(body, as: %Apus.Message{})
+
+      error ->
+        error
+    end
+  end
+
+  def handle_config(config), do: config
+
+  defp uri(config) do
+    "https://api.twilio.com/2010-04-01/Accounts/#{config.account_sid}/Messages.json"
+  end
+
+  defp headers(config) do
+    auth = Base.encode64("#{config.account_sid}:#{config.auth_token}")
+
+    [
+      Authorization: "Basic #{auth}",
+      "Content-Type": "application/x-www-form-urlencoded; charset=UTF-8"
+    ]
+  end
+
+  defp convert_to_twilio_params(message, config) do
     message
     |> Map.from_struct()
     |> maybe_put_service_sid(config)
-    |> Message.create()
-    |> format_response
+    |> Map.to_list()
   end
 
-  def handle_config(config), do: config
+  defp to_query_string(list) do
+    list
+    |> Enum.flat_map(fn
+      {key, value} when is_list(value) -> Enum.map(value, &{camelize(key), &1})
```

Comment:
Is this function needed? I'm unsure which query param this would be supporting?

Reply:
Good point, I'll remove it, we don't currently make a request that would use it.